commands_setSellAmount.go

```diff
@@ -63,5 +63,22 @@ func (ssa setSellAmountCmd) ToAuditEvent() types.AuditEvent {
 }
 
 func (ssa setSellAmountCmd) Execute() {
-	consoleLog.Warning("Not implemented: SET_SELL_AMOUNT")
+	autoTxKey := types.AutoTxKey{
+		Stock:  ssa.stock,
+		UserID: ssa.userID,
+		Action: "Sell",
+	}
+	_, found := workATXStore[autoTxKey]
+	if found {
+		// autoTx already exists, we'll need to cancel it.
```

Comment:
@trstephen There's heavy dupe across this PR. Is there a file I can stick some of the of the auxilary functions in? Might make an `autoTxUtil` file and just dump some in there.